packages/contracts/contracts/core/DAO.sol

```diff
@@ -95,6 +95,23 @@ contract DAO is
         __PermissionManager_init(_initialOwner);
     }
 
+    /// @inheritdoc PermissionManager
+    function isPermissionRestrictedForAnyAddr(bytes32 _permissionId)
```

Comment:
Please note that `PluginRepo.sol` is inheriting `PermissionManager` 
perhaps it also can declare its own restrictions, if necessary

Reply:
PluginRepo's PermissionManager is a different picture and doesn't need the role restrictions as it happens with DAO, because dao handles all of its permissions globally, where PluginRepo only handles its own... So we're good !